chrome-extension/popup.js

```diff
@@ -7,6 +7,8 @@ const COPY_FILE_URL = 'https://www.googleapis.com/drive/v3/files/fileId/copy';
 const GOOGLE_DOC_URL = 'https://docs.google.com/document/d/';
 const REVOKE_TOKEN_URL = 'https://accounts.google.com/o/oauth2/revoke?token=';
 const USER_INFO_URL = 'https://www.googleapis.com/oauth2/v1/userinfo?access_token=';
+// TODO: Change WEBAPP_URL from local URL to deployed website URL
+const WEBAPP_URL = 'https://8080-cacf7a03-5e11-4b90-94f2-e81659d32917.us-east1.cloudshell.dev';
```

Comment:
nit: I think this is a good place to use an environment variable because the value literally changes based on whether we're in dev or prod environment. I'm not 100% sure if it works in JS like it does in Node but it might just be `process.env.WEBAPP_URL`.

Reply:
I'm not sure what you mean about the environment variable, are you saying to set `WEBAPP_URL` to `process.env.WEBAPP_URL`?

Reply:
Yep, then we can set the webapp url in the environment variables in appengine-web.xml for prod and in the env_variables file for dev

Reply:
I'm not sure if that would work actually because when the chrome extension is going to be run separately from our webapp once it's on the app store.

Reply:
Ohhh I see, yeah that makes sense. Hmmm then I don't think there's a way to set the url dynamically from the chrome extension based on whether we're in prod or dev. I guess it's not too big of a deal to hardcode it for now then